public/zt.js

```diff
@@ -30,44 +32,87 @@ function generateDataElements(data, listElement,formElement) {
       element.innerHTML += `<span class="ms-auto" id="${member.config.id}-ipAssigment">${member.config.ipAssignments[0]}</span>`;
 
       listElement.append(element)
+
     }
-    else{
 
-      
+    //Generate options for form
+    if (!member.hidden) {
+      let selectElement = $('#Select-form');
+
+
+      //Checking if member has name if not assigning member id as his shown name
+      let displayName = member.name + " " + member.description;
+      if (displayName === " ") {
+        displayName = member.config.id;
+      }
+
+      //Coloring diffrent options for easier seeing (green - authorised, red - unauthorised)
+      if (member.config.authorized) {
+        selectElement.innerHTML += `<option style="color:Green" value="${[member.config.id, member.name, member.description]}">${displayName}</option>`
+      }
+      else {
+        selectElement.innerHTML += `<option style="color:Red" value="${[member.config.id, member.name, member.description]}">${displayName}</option>`
+      }
+
     }
-    
   })
 }
 
-function generateForm(unAuthorized, formElement){
-  
-  let element = document.createElement('form')
-  element.method = "post"
-  element.action = "https://api.zerotier.com/api/v1/network/0cccb752f7ccba90/member/" + unAuthorized.config.id;
-
-  element.innerHTML += `<ul>
-                          <li>
-                            <label for="name">Name:</label> <input id="name" type="text" value="name">
-                          </li>
-                          <li>
-                            <label for="description">description:</label> <input id="description" type="text" value="description">
-                          </li>
-                          <li>
-                            <label for="authorize">Authorize</label> <input id="authorize" type="checkbox" value="authorized"> 
-                          </li>
-                          <li>
-                          <input type="submit">
-                          </li>`;
-
-  formElement.append(element)
+//Generate form with network members
+function generateForm() {
+
+  const ZtForm = $('#ZT-form');
+
+  const ZTSelect = $('#Select-form');
+  unAuthorized = ZTSelect.value.split(',');
+
+  if (!$('#Post-Form')) {
+    let element = document.createElement('form');
+    element.id = "Post-Form";
+    element.innerHTML += `<ul>
```

Comment:
Maybe add some [bootstrap styling](https://getbootstrap.com/docs/5.3/forms/overview/#overview) to make this look better?